test/uk/gov/hmrc/helptosave/repo/EligibilityStatsStoreSpec.scala

```diff
@@ -36,21 +32,18 @@ class EligibilityStatsStoreSpec extends TestSupport with Eventually with MongoSu
       val document = Json.obj("eligibilityReason" -> 7, "source" -> "Digital", "total" -> 1).value
 
       "return results as expected" in {
-        withMongo { reactiveMongoComponent ⇒
-          val store = newEligibilityStatsMongoStore(reactiveMongoComponent)
+        val store = newEligibilityStatsMongoStore(reactiveMongoComponent)
 
-          Await.result(store.collection.insert(document), 5.seconds)
-          Await.result(store.getEligibilityStats(), 5.seconds) shouldBe List(EligibilityStats(Some(7), Some("Digital"), 1))
-        }
+        await(store.collection.insert(document))
```

Comment:
since its a real db instance, don't we need to clear the data before or after each test, so that tests wont screwup with existing data in mongo ?

Reply:
in `MongoSupport` the data in mongo is cleared before each test in `beforeEach`